src/app/modules/addresses/addresses.component.ts

```diff
@@ -0,0 +1,105 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { Store } from '@ngrx/store';
+import { Subject } from 'rxjs';
+import { takeUntil } from 'rxjs/operators';
+import { Address } from 'src/app/models/address.model';
+import { selectCurrentCustomerOrganization } from 'src/app/store/current-customer/current-customer.selectors';
+import { getAddressess } from './store/addresses.actions';
+import { selectOrganizationAddresses } from './store/addresses.selectors';
+import { faChevronLeft, faChevronRight } from '@fortawesome/free-solid-svg-icons';
+import { pageSize } from './addresses.constants';
+
+@Component({
+  selector: 'vc-addresses',
+  templateUrl: './addresses.component.html',
+  styleUrls: [
+    './addresses.component.scss',
+  ],
+})
+export class AddressesComponent implements OnInit, OnDestroy {
+  unsubscriber = new Subject();
+
+  curentCustomerOrganization$ = this.store.select(selectCurrentCustomerOrganization);
+
+  curentCustomerOrganizationId?: string;
+
+  addresses?: Address[] | null;
+
+  totalCount?: number | null;
+
+  cursor = '0';
+
+  page = 1;
+
+  pageSize = pageSize;
+
+  faChevronLeft = faChevronLeft;
+
+  faChevronRight = faChevronRight;
+
+  sortDescending = 'desc';
+
+  sortAscending = 'asc';
```

Comment:
May be move it to separate reusable constants file/module?

Reply:
done